### PR TITLE
[15.0][MIG] account_payment_purchase

### PR DESCRIPTION
**Pull request by koenloodts**

### CODE REVIEW

**Review by rauferdeveloper — APPROVED** [review]

**Review by rauferdeveloper — CHANGES_REQUESTED** [review]
@koenloodts Include in the change history the account_payment_mode and account_payment_partner modules that do not appear.

**Review by brendapaniagua — APPROVED** [review]
Functional review! LGTM!

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+git+https://github.com/OCA/bank-payment@refs/pull/873/head#subdirectory=setup/account_payment_partner
+git+https://github.com/OCA/bank-payment@refs/pull/870/head#subdirectory=setup/account_payment_mode
```
Comment (yajo) [inline test-requirements.txt:2]: You can remove this, as both PRs are now merged.

**Review by yajo — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
👍

**Comment by MiquelRForgeFlow** [comment]
Please, cherry-pick https://github.com/OCA/bank-payment/commit/378abcd9b45f117a855eeb9cd42d7b98f8123980.

**Comment by airlessproject** [comment]
Please cherry-pick https://github.com/airlessproject/bank-payment/commit/89d3ba54718aa06117ea71a5a81d427df0a4f0f4

Fixes unit test

**Comment by airlessproject** [comment]
@rafaelbn @brendapaniagua @yajo Please check [new PR](https://github.com/OCA/bank-payment/pull/928), since no activity on this one by the original author since it was created, and there were some fixes as well in the new one.

**Comment by sebalix** [comment]
Test:
/ocabot rebase

**Comment by pedrobaeza** [comment]
Superseded by #928

**Pull request closed without merging**